clients/stellar-relay-lib/resources/stellar_relay_config_testnet.json

```diff
@@ -0,0 +1,14 @@
+{
+  "connection_info": {
+    "address": "34.235.168.98",
+    "port": 11625
+  },
+  "node_info": {
+    "ledger_version": 19,
+    "overlay_version": 25,
+    "overlay_min_version": 23,
+    "version_str": "v19.5.0",
+    "is_pub_net": false
```

Comment:
https://stellarbeat.io/nodes/GBJQUIXUO4XSNPAUT6ODLZUJRV2NPXYASKUBY4G5MYP3M47PCVI55MNT?center=1
^ this is funny, because it says "Singapore", but the country identified is _Australia_. :laughing: